rust/src/io/writer.rs

```diff
@@ -177,6 +178,12 @@ impl<'a> FileWriter<'a> {
         }
     }
 
+    async fn write_null_array(&mut self, field: &Field, array: &ArrayRef) -> Result<()> {
+        let page_info = PageInfo::new(0, array.len());
```

Comment:
should we set `PageInfo.offset`?

Reply:
Rectified to the more accurate value, though it isn't really used while reading. Would you prefer making it an optional attribute?